bin/pd_ati_monitor.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import subprocess
+import requests
+
+'''
+Need to setup ssh no password access to 2 hosts (see below)
+
+Active Service Monitor - monitor (Browse Apps/ Custom Integration/Incoming Webhooks/)
+https://hooks.slack.com/services/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
```

Comment:
Make this URL a parameter or environment variable because it gives people access to write messages to our slack channel